test/caPutJsonLogTest.cpp

```diff
@@ -881,7 +881,7 @@ MAIN(caPutJsonLogTests)
     startIoc();
     CaPutJsonLogTask *logger =  CaPutJsonLogTask::getInstance();
     if (logger == NULL) testAbort("Failed to initialize logger.");
-    logger->initialize(logServerAddress.c_str(), caPutJsonLogOnChange);
+    logger->initialize(logServerAddress.c_str(), caPutJsonLogOnChange, 5.0);
```

Comment:
With the adjustment I described above, this change to the test code is no longer needed.

Reply:
Reverted!